docker/app/utils/checkpoint.py

```
@@ -0,0 +1,153 @@
+        model_config = checkpoint.get('model_config', {})
+
+        logging.info(f"チェックポイントを読み込みました (エポック {epoch}, 検証損失 {val_loss:.4f}, BLEU {bleu_score:.4f})")
+
+        return {
+            'epoch': epoch,
+            'best_valid_loss': val_loss,
+            'best_bleu': bleu_score,
+            'model_config': model_config
+        }
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)